automation/build-artifacts.sh

```diff
@@ -75,3 +75,9 @@ ansible-test sanity
 /usr/local/bin/ansible-lint roles/* -x 204
 
 cd $ROOT_PATH
+
+# If PR changed something in ./plugins or ./roles it is required to have changelog
+if [[ $(git diff --quiet FETCH_HEAD ./plugins ./roles)$? -eq 1 && $(git diff --quiet FETCH_HEAD ./changelogs)$? -eq 0 ]]; then
+        echo "Please add changelog.";
+        exit 1;
```

Comment:
Does it mean that for every change in plugins or roles we will have CI complain for missing changelog?

Reply:
Is there an easy way how to figure out if there was a change adding some options/parameters or a new modules?